YunoHost-Apps/gitea_ynh
# Manifestv2 improvements

## Problem

- V1
- Mail stack not tested
- Mail reply not configured
- Group support not configured
- #76 
- #87 

## Solution

- Upgrade package in v2
- Test and fix config for email stack
- Add support for mail reply
- Add support for group and team mapping
- Fix issues

Note that I've decided to temporately disable the SSO feature untils https://github.com/go-gitea/gitea/pull/18452 is merged (as we can't chose booth SSO and LDAP currently and for now we miss many feature if we don't use LDAP).

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)
